distributed/cli/dask_worker.py

```diff
@@ -295,23 +305,27 @@ def main(
         if v is not None
     }
 
-    if nprocs == "auto":
-        nprocs, nthreads = nprocesses_nthreads()
+    if nprocs is not None:
+        warnings.warn("The --nprocs flag has been renamed to --num-workers. ")
```

Comment:
Since this is a deprecation warning, we should probably raise a `FutureWarning` instead of a `UserWarning`

Reply:
Let's also mention that `--nprocs` will be removed in a future release in the warning message